bfcfg

```diff
@@ -1157,6 +1158,21 @@ misc_cfg()
       rm -f ${tmp_file}
       log_msg "misc: BF_BUNDLE_VERSION=${BF_BUNDLE_VERSION}"
     fi
+
+    if [ -n "${BMC_UPGRADE_RESET}" ]; then
+      printf "\\x07\\x00\\x00\\x00" > ${tmp_file}
+      if [ "${BMC_UPGRADE_RESET}" == "0" ]; then
+        printf "\\x00" >> ${tmp_file}
```

Comment:
Q: what is the use case for `BMC_UPGRADE_RESET = 0` ? Does it make sense to support it?